components/NavBar/NavBar.module.css

```
@@ -7,10 +7,31 @@
   top: 0px;
   height: 60px;
 }
+.mobileTopNavContainer {
+  width: 100%;
+  background-color: #fff;
+  border-bottom: 1px solid #dbdbdb;
+  position: fixed;
+  left: 0;
+  z-index: 2;
+  top: 0px;
+  height: 44px;
+}
+.mobileBottomNavContainer {
+  width: 100%;
+  background-color: #fff;
+  border-top: 1px solid #dbdbdb;
+  position: fixed;
+  left: 0;
+  z-index: 2;
+  top: 96%;
```

Top 96% means the element height is 4%. Is it? 

You should do
```suggestion
  bottom: 0;
```